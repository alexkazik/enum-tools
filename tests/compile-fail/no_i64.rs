#![no_implicit_prelude]

use ::enum_tools::EnumTools;

#[derive(EnumTools, Debug, Clone, Copy, Eq, PartialEq)]
#[repr(u64)]
pub(crate) enum EG {
    #[enum_tools(rename = "A*")]
    A = 0,
    B = 9,
    C = 2,
    D = 0xf000_0000_0000_0000,
}

fn main() {}
