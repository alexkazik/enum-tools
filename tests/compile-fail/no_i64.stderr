error: can't be parsed as i64
  --> tests/compile-fail/no_i64.rs:12:5
   |
12 |     D = 0xf000_0000_0000_0000,
   |     ^
